README.md

```diff
@@ -82,29 +82,34 @@ $ spotirec
 You can pass arguments to specify the what the recommendations should be based on - these are mutually exclusive
 ```
 $ spotirec -t 
+$ spotirec -st
 $ spotirec -a
 $ spotirec -tc
+$ spotirec -stc
 $ spotirec -ac
 $ spotirec -gc
 $ spotirec -gcs
 $ spotirec -c
 ```
 where
 - `-t` is based off your 5 most played tracks,
+- `-st` is based off your 5 most recently liked tracks,
 - `-a` is based off your 5 most played artists,
 - `-tc` you can define 1-5 of your most played tracks,
+- `-stc` you can define 1-5 of your 50 most recently liked tracks,
 - `-ac` you can define 1-5 of your most played artists,
 - `-gc` you can define 1-5 of your most played valid seed genres,
-- `-gcs` you can define 1-5 preset genre seeds,
+- `-gcs` you can define 1-5 preset genre seeds, and
```

Comment:
Seems like something's missing

Reply:
Could you clarify?

Reply:
~~`, and` with no following sentence. It doesn't seem to mesh with the following sentence~~ ah, just as I wrote the reply, I saw :upside_down_face: